composeApp/src/commonMain/kotlin/org/ooni/probe/ui/dashboard/DashboardViewModel.kt

```diff
@@ -73,7 +77,19 @@ class DashboardViewModel(
             TaskSettings(
                 name = "web_connectivity",
```

Comment:
We may want to start defining various test "classes" as some tests have different keys.

Reply:
Same reply https://github.com/ooni/probe-multiplatform/pull/30#discussion_r1703870974